src/kirby/components/page/page.component.ts

```diff
@@ -175,13 +177,18 @@ export class PageComponent implements OnInit, OnDestroy, AfterViewInit {
         this.onEnter();
       }
     });
+
+    addEventListener(tabClicked, () => {
```

Comment:
What does this method refer to? If it's window.addEventListener it should be explicit...

Reply:
Yes, and maybe use window abstraction